getconf/base.py

```diff
@@ -169,6 +170,19 @@ def gettimedelta(self, key, default='0d', doc=''):
         else:
             return datetime.timedelta(**{unit: value})
 
+    def getpath(self, key, default=Path(), doc=''):
+        assert (
+            default is None or isinstance(default, Path)
+        ), 'getpath("%s", %s) has an invalid default value type.' % (key, repr(default))
+        value = self._get(key, default=default, doc=doc, type_hint='Path')
+        if value is None:
+            return None
```

Comment:
Actually not sure about the usage for None, I just followed the pattern.

Reply:
Yes, I'd say that's what would be expected.